src/handler/http/request/search/mod.rs

```diff
@@ -245,27 +232,16 @@ pub async fn around(
     let query = web::Query::<HashMap<String, String>>::from_query(in_req.query_string()).unwrap();
     let stream_type = match get_stream_type_from_request(&query) {
         Ok(v) => v.unwrap_or(StreamType::Logs),
-        Err(e) => {
-            return Ok(HttpResponse::BadRequest().json(MetaHttpResponse::error(
-                http::StatusCode::BAD_REQUEST.into(),
-                e.to_string(),
-            )))
-        }
+        Err(e) => return Ok(bad_request(e)),
     };
 
     let around_key = match query.get("key") {
         Some(v) => v.parse::<i64>().unwrap_or(0),
-        None => {
-            return Ok(HttpResponse::BadRequest().json(MetaHttpResponse::error(
-                http::StatusCode::BAD_REQUEST.into(),
-                "around key is empty".to_string(),
-            )))
-        }
-    };
-    let around_size = match query.get("size") {
-        Some(v) => v.parse::<usize>().unwrap_or(0),
-        None => 10,
+        None => return Ok(bad_request("around key is empty")),
     };
+    let around_size = query
+        .get("size")
+        .map_or(10, |v| v.parse::<usize>().unwrap_or(0));
```

Comment:
@oasisk Is default value 10 correct? If it is, why don't we use it when parsing fails?

Reply:
yes, we can use it.